packages/live-preview-sdk/src/csm/index.ts

```diff
@@ -0,0 +1,74 @@
+import { vercelStegaEncode } from '@vercel/stega';
+import { debug } from '../helpers';
+import jsonPointer from 'json-pointer';
+import { EntitySource, GraphQLResponse, Source } from './types';
+
+export class ContentSourceMaps {
```

Comment:
Wondering if we need the class as it has no constructor, internal data or something like that and only exposes one public function

We could also use only functions and only export the relevant one

Reply:
Sure that's good for me, I went with a class since the other files are also using that approach